src/bin/huak/test_utils.rs

```diff
@@ -0,0 +1,77 @@
+use std::{
+    env::current_dir,
+    fs::{copy, create_dir_all, File},
+    path::{Path, PathBuf},
+};
+use zip::ZipArchive;
+/// unzips a project from a given zip file into the provided directory.
+/// this command only works the CURRENT DIRECTORY is unchchanged.\
+/// resource_archive: PathBuf - directory, relative to resources.
+pub fn create_py_project_sample(resource_archive: &PathBuf, target_directory: &PathBuf) -> bool {
+    if !Path::new(resource_archive).is_file() {
+        println!("resource archive does not exist");
```

Comment:
```suggestion
        panic!("resource archive does not exist");
```
Should we panic?

Reply:
Those are actually for debugging purposes- you will get a panic if it's not there.